modules/common/math/kalman_filter_1d.cc

```diff
@@ -40,7 +40,7 @@ bool KalmanFilter1D::Init(const float& x) {
 
   Eigen::Matrix<float, 2, 2> q;
   q.setIdentity();
-  q *= 2.0f;
+  q *= 10.0f;
```

Comment:
change it to load config instead of directly hard code the coefficient here.

Reply:
yes, will fix later in a separate CR